cmd/lingo/main.go

```diff
@@ -73,10 +74,14 @@ func run() error {
 		// MessengerURLs is a list of (comma-separated) URLs to listen for requests and send responses on.
 		//
 		// Format: <request-subscription1>|<response-topic1>,<request-subscription2>|<response-topic2>,...
+		// You can optionally also specify the max number of handlers for each messenger URL pair.
+		// If not specified, the default is 1000.
+		// Format with setting max concurrency: <request-subscription1>|<response-topic1>|<max-handlers1>,
```

Comment:
Slight opinion: In my head, the `|` character was used to symbolize a linux pipe. It is a little confusing to tack on options with additional pipes, what about other settings? Will they be added with more `|`'s? I would prefer something more like: `<request-sub>|<response-topic>,concurrency:1000`. Longer term I think we might need to move to a yaml/json config file.

Reply:
Another option would be to specify a global value in the controller that applies for the entire controller. It could either be implemented as `CONCURRENCY_PER_MESSENGER` or `MESSENGER_TOTAL_CONCURRENCY`

Reply:
I was wondering why the `|` and did not think about it as a pipe but that does make sense! My mind is very context based. I only think in pipes when I'm in my shell.

I prefer to keep as-is for now since the default of 1000 is probably good enough for most people.